.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 .DS_Store
-client/dist/bundle.js
\ No newline at end of file
+client/dist/bundle.js
+newrelic_agent.log
```

Comment:
you might want to put your NR key file in here just as a preventative measure

Reply:
I've stored my key in the parent folder for this repo, so it won't ever be accidentally committed.